app/survey_config/survey_config.py

```diff
@@ -21,7 +23,7 @@ class SurveyConfig:
     copyright_text: Optional[LazyString] = lazy_gettext(
         "Use of address data is subject to the terms and conditions."
     )
-    base_url: str = DEFAULT_EN_BASE_URL
+    base_url: Optional[str] = ACCOUNT_SERVICE_BASE_URL
```

Comment:
This should always exist, i.e not optional as you have a default.

Reply:
Remove these Optional hints, thanks!